### ISSUE TITLE
[Android] Background image not filling the entire container.

### ISSUE THREAD

**Issue by Gunashree-R-V** [microsoft/AdaptiveCards#4632]
# Platform

* Android

# Author or host

Microsoft Teams

# Version of SDK

1.2.10

# Details

When a container has a background image, the entire container is not filled. And elements within the container are not aligned as observed on Desktop.

Here are the screenshots of experience comparison:

| Desktop | Android | 
|-----------|:-----------:|
| ![image](https://user-images.githubusercontent.com/28745768/90759605-49720080-e2fe-11ea-9df4-f9a0d4c7b518.png)| ![image](https://user-images.githubusercontent.com/28745768/90759621-4ecf4b00-e2fe-11ea-9544-8e35477818c9.png)| 

Here is another broken experience:

| Desktop | Android | 
|-----------|:-----------:|
|![image](https://user-images.githubusercontent.com/28745768/90760512-9efadd00-e2ff-11ea-9372-81e9c5fcac85.png)| ![image](https://user-images.githubusercontent.com/28745768/90760528-a7ebae80-e2ff-11ea-81e8-37550d4d299e.png)| 

Attaching the card json.
[Jsons.zip](https://github.com/microsoft/AdaptiveCards/files/5102409/Jsons.zip)

cc: @VikrantSingh01, @anbah34, @shbhadra.

**Comment by almedina-ms** [comment]
@shalinijoshi19 , I think this is a by design issue. Containers are rendered with a padding unless they have the bleed property assigned as true. For bleed to work, the container must have a different style than it's parent. Probably @matthidinger could clarify the default behavior of containers but I was able to have a work around that seems more like the expected behaviour

![image](https://user-images.githubusercontent.com/35784165/91224268-b27abf00-e6d6-11ea-8487-b7246c917eb1.png)


```json
{
    "type": "AdaptiveCard",
    "bleed": true,
    "body": [
        {
            "items": [
                {
                    "text": "Test Card.",
                    "id": "txtid",
                    "type": "TextBlock"
                },
                {
                    "placeholder": "Enter your text",
                    "value": "default text",
                    "id": "newtxt",
                    "type": "Input.Text"
                },
                {
                    "text": "My new textblock",
                    "id": "resd",
                    "type": "TextBlock"
                },
                {
                    "value": "",
                    "id": "dateid",
                    "type": "Input.Date"
                },
                {
                    "choices": [
                        {
                            "title": "Choice 1",
                            "value": "Choice 1"
                        },
                        {
                            "title": "Choice 2",
                            "value": "Choice 2"
                        }
                    ],
                    "value": "mychoice",
                    "id": "idchoice",
                    "type": "Input.ChoiceSet"
                },
                {
                    "text": "text3",
                    "type": "TextBlock"
                },
                {
                    "text": "Text4",
                    "type": "TextBlock"
                },
                {
                    "text": "Text5",
                    "type": "TextBlock"
                }
            ],
            "backgroundImage": "https://wallpaperaccess.com/full/2849664.jpg",
            "bleed": true,
            "style": "good",
            "type": "Container"
        }
    ],
    "version": "1.0"
}
```

**Comment by almedina-ms** [comment]
issue fixed by #4956
